packages/manager/cypress/e2e/core/linodes/create-linode-view-code-snippet.spec.ts

```diff
@@ -0,0 +1,206 @@
+            .click();
+
+          // Validate Ansible and links
+          ui.autocomplete.find().click();
+
+          ui.autocompletePopper
+            .findByTitle('Ansible')
+            .should('be.visible')
+            .click();
+          cy.contains(
+            'a',
+            'Getting Started With Ansible: Basic Installation and Setup'
+          ).should('be.visible');
+          cy.contains('a', 'Linode Cloud Instance Module').should('be.visible');
+          cy.contains('a', 'Manage Personal Access Tokens').should(
+            'be.visible'
+          );
+          cy.contains('a', 'Best Practices For Ansible').should('be.visible');
+          cy.contains(
+            'a',
+            'Use the Linode Ansible Collection to Deploy a Linode'
+          ).should('be.visible');
+
+          // Validate Terraform and links
+          ui.autocomplete.find().click();
+          ui.autocompletePopper
+            .findByTitle('Terraform')
+            .should('be.visible')
+            .click();
+          cy.contains('a', `A Beginner's Guide to Terraform`).should(
+            'be.visible'
+          );
+          cy.contains('a', 'Install Terraform').should('be.visible');
+          cy.contains('a', 'Manage Personal Access Tokens').should(
+            'be.visible'
+          );
+          cy.contains('a', 'Use Terraform With Linode Object Storage').should(
+            'be.visible'
+          );
+          cy.contains(
+            'a',
+            'Use Terraform to Provision Infrastructure on Linode'
+          ).should('be.visible');
+          cy.contains(
+            'a',
+            'Import Existing Infrastructure to Terraform'
+          ).should('be.visible');
+
+          // Validate SDK's tab
+          ui.tabList.findTabByTitle(`SDK's`).should('be.visible').click();
+
+          ui.autocomplete.find().click();
+
+          // Validate linodego and links
+          ui.autocompletePopper
+            .findByTitle('Go (linodego)')
+            .should('be.visible')
+            .click();
+          cy.contains('a', 'Go client for Linode REST v4 API').should(
+            'be.visible'
+          );
+          cy.contains('a', 'Linodego Documentation').should('be.visible');
+
+          ui.autocomplete.find().click();
+
+          // Validate Python API4 and links
+          ui.autocompletePopper
+            .findByTitle('Python (linode_api4-python)')
+            .should('be.visible')
+            .click();
+
+          cy.contains(
+            'a',
+            'Official python library for the Linode APIv4 in python'
+          ).should('be.visible');
+          cy.contains('a', 'linode_api4-python Documentation').should(
+            'be.visible'
+          );
+
+          ui.button
+            .findByTitle('Close')
+            .should('be.visible')
+            .should('be.enabled')
+            .click();
+        });
+    });
+  });
+  describe('Create Linode flow with apicliDxToolsAdditions enabled', () => {
```

Comment:
```suggestion
  describe('Create Linode flow with apicliDxToolsAdditions disabled', () => {
```
😅 my bad again for missing this before